packages/perseus/src/widgets/interactive-graphs/graphs/components/angle-indicators.tsx

```diff
@@ -80,13 +77,8 @@ export const PolygonAngle = ({
         ) : null;
     }
 
-    // Midpoint betwen ends of arc
-    const isOutside = shouldDrawArcOutside(
-        [x3, y3],
-        centerPoint,
-        range,
-        polygonLines,
-    );
+    // Note: Need to pass in endpoints in a clockwise order for the cross product.
+    const isOutside = shouldDrawArcOutsidePolygon(centerPoint, endPoints);
```

Comment:
Just want to double-check if other uses of `shouldDrawArcOutside` should use this new method as well or if the ray-tracing method is still best for them?

Reply:
Here's the screenshot of the angle being broken

<img width="284" alt="image" src="https://github.com/user-attachments/assets/42c42a1a-b303-4a30-82c1-8ef58df26265" />